### DISCUSSION TITLE
I am looking for a react-select alternative including virtual lists

### DISCUSSION THREAD

**Discussion by wibed** [petyosi/react-virtuoso#628] [Q&A] [answered]
has react virtuoso a "select" functionality ? similar to react-select loading the dropdown asyncronously.

**Comment by petyosi** [comment, marked as answer]
No, no built-in select functionality. You can maintain the state by yourself, this example is a nice starting point, as it includes keyboard nav: https://virtuoso.dev/keyboard-navigation/
